.travis.yml

```diff
@@ -43,12 +49,11 @@ jobs:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
     - stage: "Additional Tests"
-      env: EMBER_TRY_SCENARIO=ember-lts-2.16
```

Comment:
does that mean Ember 2.16 is no longer supported? in that case this probably should be considered a breaking change and cause a new major release 🤔